dashboard/test/ui/utils/selenium_browser.rb

```diff
@@ -14,7 +14,17 @@ def self.local(browser: :chrome, headless: true)
       options[:options] = Selenium::WebDriver::Firefox::Options.new
       options[:options].headless! if headless
       options[:options].add_argument('window-size=1280,1024')
+    when :edge
+      options[:options] = Selenium::WebDriver::Edge::Options.new
+      options[:options].headless! if headless
+      options[:options].add_argument('window-size=1280,1024')
```

Comment:
Should we reverse the logic and not add this in case of safari?

Reply:
....or maybe we want this for safari too? 🤷

Reply:
We don't test safari locally. Though, that would be nice. Not sure exactly what you are advocating, however, Reversing what logic?